src/style.css

```diff
@@ -1 +1,79 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Book.ttf'), url('fonts/Avenir-Heavy.ttf');
```

Comment:
It's recommended to specify different font weights with separate @font-face declarations for each weight. Here, you have combined Avenir-Book and Avenir-Heavy into one declaration, which may not work as intended.